### ISSUE TITLE
"Shallow" GUI refactor for improved usability

### ISSUE THREAD

**Issue by talonchandler** [mehta-lab/recOrder#225]
@mattersoflight and I met today and discussed several changes to `recOrder`'s GUI. While most of these changes should be fairly shallow, I predict based on experience with `recOrder` that these changes will require another complete round of testing, debugging, and likely iteration---hence the delay to 0.2.1. 

Briefly, we discussed:
- Eliminate "Acquisition / Reconstruction" nested tabs by moving the "Processing" tab into the top set of tabs
- Group the "Z padding" parameter together with the "Phase reconstruction" parameters (specifically, the regularization parameter)
- Move "Action" buttons to the top of every tab --- specifically, the "Reconstruct" and "STOP" should be at the top of the offline "Acquisition / Reconstruction" tab. 
- Moving or mirroring some of the microscope parameters in the "Calibration" and "Acquisition / Reconstruction" tabs --- on thinking more I don't think this is the best solution---see #216 for more context.

**Comment by ziw-liu** [comment]
APIs in 0.x versions are not really bound by the general semantic versioning rules, but I feel that a refactoring of the GUI layout does qualify for a minor version number increment instead of a patch.

**Comment by talonchandler** [comment]
Roger that. Let's leave it labelled at 0.2.1 (partially to encourage ourselves to constrain the scope) and we can rename to 0.3 at release time if necessary.

**Comment by mattersoflight** [comment]
It makes sense to release 0.2.0 with the current level of testing of the Online mode and release 0.2.1 (or 0.3) after significant testing of the GUI, especially the offline mode. 

I am collecting the bugs in offline mode as a task list here:
- [ ] label `Calibration Metadata` field as Required. It is currently labeled Optional, which leads to an error.
![image](https://user-images.githubusercontent.com/2934183/193981533-1e9cc57d-52e5-4dcf-b795-6f1508205e68.png)

**Comment by mattersoflight** [comment]
Another suggestion to reach a functional offline mode: 
In 0.3.0, hide the offline mode UI completely, and write reconstruction methods that can be invoked from within napari console after importing recOrder. These methods/scripts can read the data/configs and populate reconstructions in the same napari window. I can see this being a functional mode for the majority of our users after they have been shown how to use napari’s python console.

**Comment by talonchandler** [comment]
> label Calibration Metadata field as Required. It is currently labeled Optional, which leads to an error.

Thanks for flagging this label --- I just removed it in the `0.2.0-integration` branch. I don't think "Calibration Metadata" is actually required (for PhaseFromBF, for example), but it definitely isn't always optional. 

I think this points towards deeper changes that are planned for 1.0.0 --- this field should only appear when you're reconstruction things that actually require this data. 

> Offline reconstruction in QLIPP and 3D mode sometimes results in zero-valued retardance and BF channels.
I tested this with 2T_3P_16Z_128Y_256X_Kazansky_1 dataset.

I've also seen this. Sometimes `recOrder` confusingly sets the napari channel slides to the wrong position when there are existing layers which makes things appear black. 

I've also seen offline mode genuinely show zeros. Anecdotally, I've had better luck loading .ome.tiffs in offline mode than .zarrs, so there might be an issue with the `wo` reader here.
